src/plugins/bibliography/bibliography-utils.ts

```diff
@@ -99,6 +100,24 @@ export const buildDecorations = (
     })
   }
 
+  // Trigger bibliography element update
+  if (triggerUpdate) {
+    doc.descendants((node, pos) => {
+      if (isBibliographyElement(node)) {
+        decorations.push(
+          Decoration.node(
+            pos,
+            pos + node.nodeSize,
+            {},
+            {
+              triggerUpdate: true,
```

Comment:
I'm a bit perplexed by the triggerUpdate usage. Is it only present to make prosemirror trigger updates while no actual changes are made in the dom tree?

Reply:
Yes, I had to do that to trigger the bibliography element `updateContents`.

I cannot communicate from the plugin to the bibliography element node without actually changing the node. So I added a node decoration from the plugin on the bibliography element node (by putting the `triggerUpdate ` into the spec argument) to trigger the update.

Reply:
I see, can you please add a comment somewhere in the code where "triggerUpdate" is referenced, what is it for. While the name is pretty self-explanatory, this is still an unusual way to make things work in PM. Thanks!

Reply:
I updated the comment, adding more details. Please check and let me know if it's clear.